### ISSUE TITLE
ErrorHandler#getShowMessageInTitle() is ignored

### ISSUE THREAD

**Issue by razisam** [eclipse/jetty.project#12361]
**Jetty version(s)**
12.0.9

**Jetty Environment**
ee9

**Java version/vendor** `(use: java -version)`
JDK21

**OS type/version**
Windows & Linux

**Description**
the method `org.eclipse.jetty.ee9.nested.ErrorHandler#getShowMessageInTitle` is not called within `org.eclipse.jetty.ee9.nested.ErrorHandler#writeErrorPageHead`. actually i can't find any usages of `getShowMessageInTitle()` and why is is called `getShowMessageInTitle`? shouldn't it be `isShowMessageInTitle()` like the two others (`isShowServlet` and `isShowStacks`)

**Comment by razisam** [comment]
tried to configure `org.eclipse.jetty.ee9.nested.ErrorHandler` like this, to get rid of too many information in error page.

```
<?xml version="1.0"  encoding="UTF-8"?>
<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
 <Configure class="org.eclipse.jetty.ee9.webapp.WebAppContext">
  <Set name="extractWAR">true</Set>
  <Set name="copyWebDir">true</Set>
.....
  <Get name="ErrorHandler">
    <Call name="setShowMessageInTitle">
      <Arg type="boolean">false</Arg>
    </Call>
    <Call name="setShowServlet">
      <Arg type="boolean">false</Arg>
    </Call>
    <Call name="setShowStacks">
      <Arg type="boolean">false</Arg>
    </Call>
  </Get>

</Configure>
```

**Comment by razisam** [comment]
org/eclipse/jetty/ee9/nested/ErrorHandler.java:372

```
        if (getShowMessageInTitle() && message != null && !message.equals(status))
        {
            writer.write(' ');
            writer.write(StringUtil.sanitizeXmlString(message));
        }
        writer.write("</title>\n");
```

**Comment by janbartel** [comment]
@razisam you're right that the `writeErrorPageHead` method should call `getShowMessageInTitle()`, and that it should be called `isShowMessageInTitle()`. However, I've checked back to even jetty 9.4 and the method is called this, and it's value is ignored. In other words, it's never worked! And in jetty-12.0 ee8 and ee9 we aim to be bug-for-bug compatible with prior versions of jetty.  The bug appears to be also in ee10, so could probably be fixed there. Note that in jetty-12.1 for the ee11 `ErrorHandler`, the bug(s) are fixed because it inherits off core `ErrorHandler`, and I already made those changes.
